client-app/broadcast.ts

```diff
@@ -105,4 +107,11 @@ export function setupBroadcastGlobalListeners() {
       location.href = `/change-password?returnUrl=${pathname + search + hash}`;
     }
   });
+
+  on(dataChangedEvent, () => {
+    notifications.warning({
+      duration: DEFAULT_NOTIFICATION_DURATION,
```

Comment:
Is there a way to make it persist until user close (or actually refresh the page)?
Because it's going to be shown for 15 seconds and then be hidden - user can miss this notification.